enhanced-resolve/lib/Storage.d.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="node" />
+import { CommonFileSystemMethod } from './common-types';
+
+declare class Storage {
+    duration: number;
+    running: Object;
+    data: Object;
```

Comment:
{} is fine. It also has nothing to do with `Tapable`. But the type seems better.